### PR TITLE
Make use of selector for plot viewer page

### LINKED ISSUE #124: Adapt Plot viewer to use parameter selector
To make selections easier we should add selector to plot viewer.

### PR DESCRIPTION
**Pull request by xjules**
Resolves #124 

- provides support for multiple selectors on a single page.
- adjust the current controllers to work with parameter-selector

TODO:
- [x] make trigger if ensemble selection changes
- [x] tests
- [x] toggle visibility of selector by a button

### CODE REVIEW

**Comment by xjules** [comment]
Screenshot:
image

File: ertviz/views/selector_view.py [hunk 1]
```diff
@@ -16,32 +16,44 @@ def parameter_selector_view(parent, data_type="parameter"):
...
+            html.Div(
+                wcc.Select(
+                    id=parent.uuid(f"parameter-selector-multi-{suffix}"),
+                    multi=True,
+                    size=10,
+                    persistence="session",
+                    className="ert-dropdown",
+                ),
+                id=parent.uuid(f"container-parameter-selector-multi-{suffix}"),
+                # style={"display": "block"},
```
Comment (oysteoh) [inline ertviz/views/selector_view.py:43]: could this be removed - or made more meaningful for others if it should stay as a future usage or something?

**Comment by oysteoh** [comment]
I cannot get the toggling of the selection on/off to work - neither searching on keys. If i click manually and scroll it works as before.

**Review by oysteoh — CHANGES_REQUESTED** [review]
Make search case-insensitive and squash your commits - then i'll think we are good to go ! 👍

**Review by oysteoh — CHANGES_REQUESTED** [review]
Now it is also case-insensitive ! Great! Then please squash 👍 🚀

**Review by oysteoh — APPROVED** [review]

**Pull request merged**